python/hail/methods/__init__.py

```diff
@@ -1,8 +1,11 @@
 from .family_methods import trio_matrix
-from .statgen import linreg, ld_matrix
-from .qc import sample_qc
+from .statgen import linreg, ld_matrix, pca, hwe_normalized_pca
+from .qc import sample_qc, variant_qc
```

Comment:
Weird, I don’t see variant_qc in methods in the artifacts for this PR, only sample_qc:
https://ci.hail.is/repository/download/HailSourceCode_HailMainline_BuildDocs/50852:id/www/docs/stable/methods/index.html

Any idea what's going on?

Reply:
I definitely forgot to add it to the rst file. Unfortunately, we need to add module-level methods explicitly.